mobilecoind-dev-faucet/src/slam/tx_submitter.rs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use displaydoc::Display;
+use mc_attest_verifier::{MrSignerVerifier, Verifier, DEBUG_ENCLAVE};
+use mc_common::logger::{log, o, Logger};
+use mc_connection::{
+    Error as ConnectionError, HardcodedCredentialsProvider, RetryError, RetryableUserTxConnection,
+    SyncConnection, ThickClient,
+};
+use mc_transaction_core::{tx::Tx, validation::TransactionValidationError};
+use mc_util_uri::{ConnectionUri, ConsensusClientUri};
+use std::{iter::empty, sync::Arc};
+
+type ConnectionType = SyncConnection<ThickClient<HardcodedCredentialsProvider>>;
+
+pub struct TxSubmitter {
+    #[allow(dead_code)]
+    uris: Vec<ConsensusClientUri>,
```

Comment:
Why store this if it goes unused?

Reply:
i think i thought it might be used later, if we want to log this in some particular error message or something. but it never came about I guess

fine to delete it, someone can add it later pretty easily if they need it